Matlab_Scripts/5_Depth_Colourless/Pilot/master.m

```diff
@@ -1,14 +1,13 @@
-% This script either runs preprocesses or analysis, for the appropriate
-% condition & event type depending on its settings
-% change preprocessing parameters here
+% You run and change parameters for preprocessing and analysis here, you
+% also specify the target condition & event type
 Wrapper_VR_P3_Depth_Colourless_Pilot
 
 % This script renames triggers appropriately, refer to markerinfo.txt
 % and initializes structures in accordance with wrapper settings above
-Preprocessing_VR_P3_Depth_Colourless
+Preprocessing_VR_P3_Depth_Colourless(exp)
 
 % Loads data for use with ERP (we are not using time-frequency analysis)
-Analysis_VR_P3_Depth_Colourless
+Analysis_VR_P3_Depth_Colourless(exp,anal)
```

Comment:
have to define exp and anal in this file to be able to run it on its own

Reply:
in other words, this file should be able to be run by itself with no other user input,